src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -27,29 +27,32 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     this.updateUrl(namespaces);
 
     // sync with local-storage & url-search-params
-    reaction(() => this.contextNs.toJS(), namespaces => {
-      this.storage.set(namespaces);
-      this.updateUrl(namespaces);
-    });
+    reaction(
+      () => this.contextNs.toJS(),
+      (namespaces) => {
+        this.storage.set(namespaces);
+        this.updateUrl(namespaces);
+      }
+    );
   }
 
   getContextParams(): Partial<IQueryParams> {
     return {
-      namespaces: this.contextNs
-    }
+      namespaces: this.contextNs,
+    };
   }
 
   protected updateUrl(namespaces: string[]) {
-    setQueryParams({ namespaces }, { replace: true })
+    setQueryParams({ namespaces }, { replace: true });
   }
 
   protected async loadItems(namespaces?: string[]) {
     if (!isAllowedResource("namespaces")) {
       if (namespaces) return namespaces.map(this.getDummyNamespace);
-      return []
+      return [];
     }
     if (namespaces) {
-      return Promise.all(namespaces.map(name => this.api.get({ name })))
+      return Promise.all(namespaces.map((name) => this.api.get({ name })));
```

Comment:
Please adjust your editor's auto-format to not create  such diff-changes.

Reply:
It would be nice to include format configure file in the project, such as `.prettierrc`.